src/generic/processing-notification/index.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { Badge, Icon } from '@edx/paragon';
+import { Settings as IconSettings } from '@edx/paragon/icons';
+
+const ProcessingNotification = ({ isShow, title }) => (
```

Comment:
Can you create file with `constants.js` to store state of this notification and use it like title

```
export const NOTIFICATION_MESSAGES = {
  saving: 'saving',
  duplicating: 'duplicating',
};
```

`title: PropTypes.oneOf(Object.values(NOTIFICATION_MESSAGES)).isRequired,`

Reply:
Added to the file with global constants that we use